src/gameinfo.cpp

```diff
@@ -87,29 +87,29 @@ std::vector<std::string> Game::to_lines() {
   return lines;
 }
 
-std::vector<std::string> GameWithPickedMoves::to_lines() {
+std::vector<std::string> GameWithSampledMoves::to_lines() {
   auto lines = Game::to_lines();
 
   std::stringstream ss;
-  ss << "MovesPicked" << std::endl;
+  ss << "MovesSampled" << std::endl;
 
-  for (unsigned index : this->picked_moves_indexes) {
+  for (unsigned index : this->sampled_moves_indexes) {
     move_into_stringstream(this->moves[index], index + 1, ss);
   }
   stringstream_into_lines(ss, lines);
 
   return lines;
 }
 
-bool GameWithPickedMoves::from_lines(std::vector<std::string> lines) {
-  std::vector<std::string> game_lines, picked_moves_lines;
+bool GameWithSampledMoves::from_lines(std::vector<std::string> lines) {
+  std::vector<std::string> game_lines, sampled_moves_lines;
 
-  bool picked_moves = false;
+  bool sampled_moves = false;
   for (auto line : lines) {
-    if (line == "MovesPicked") {
```

Comment:
Hey, I think something like below would be more efficient and easier to understand. Might seem pedantic :D but I think such small improvements in the coding style will have long term benefits.
```
while line != 'MovesPicked':
    game_lines.push_back(line)
// insert all remaining lines to picked_move_lines

// Why?
// 1. all picked move lines inserted at once. I hope there is something for this in the vector api.
// 2. no 'picked_moves = True' check needed for all the picked move lines
```